From 74c8c9eb30292fb705a14cc77d9f205111ca5a93 Mon Sep 17 00:00:00 2001
From: Anupam Srivastava <ansrivastava@corp.ebay.com>
Date: Sat, 2 May 2015 13:56:53 +0530
Subject: [PATCH 09/15] Button Width

---
 fvwm/builtins.c | 15 +++++++++++++++
 1 file changed, 15 insertions(+)

diff --git a/fvwm/builtins.c b/fvwm/builtins.c
index ce22c42..d26b122 100644
--- a/fvwm/builtins.c
+++ b/fvwm/builtins.c
@@ -510,6 +510,21 @@ static void do_title_style(F_CMD_ARGS, Bool do_add)
 			if (action)
 				action += next;
 		}
+		else if (!do_add && StrEquals(parm,"buttonwidth"))
+		{
+			int width = 0;
+			int next = 0;
+
+			sscanf(action, "%d%n", &width, &next);
+
+			if (decor->button_width != width)
+			{
+				decor->button_width = width;
+				decor->flags.has_changed = 1;
+			}
+			if (action)
+				action += next;
+		}
 		else if (!do_add && StrEquals(parm,"MinHeight"))
 		{
 			int height = 0;
-- 
2.0.5

