From dac2000d01d90762ace1243b1a6719c60f7c1ede Mon Sep 17 00:00:00 2001
From: Anupam Srivastava <ansrivastava@corp.ebay.com>
Date: Sat, 2 May 2015 14:43:14 +0530
Subject: [PATCH 12/13] First Item Under Pointer

---
 fvwm/menus.c | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/fvwm/menus.c b/fvwm/menus.c
index 96fcfad..3838969 100644
--- a/fvwm/menus.c
+++ b/fvwm/menus.c
@@ -3296,6 +3296,13 @@ static int pop_menu_up(
 	}
 	context = (*pexc)->w.wcontext;
 
+	/* Popup menu with first item (not title) under pointer */
+	if(MI_IS_TITLE(MR_FIRST_ITEM(mr)))
+	{
+		y-=MI_HEIGHT(MR_FIRST_ITEM(mr));
+		do_warp_to_item = True;
+	}
+
 	/*
 	 * Create a new menu instance (if necessary)
 	 */
-- 
2.3.6

