From c7be8bceb53ac9cf4bbcf26c3defb92508bdd304 Mon Sep 17 00:00:00 2001
From: Anupam Srivastava <ansrivastava@corp.ebay.com>
Date: Sat, 2 May 2015 14:44:02 +0530
Subject: [PATCH 15/15] This Geometry Proxy

---
 libs/defaults.h               | 2 +-
 modules/FvwmProxy/FvwmProxy.c | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/libs/defaults.h b/libs/defaults.h
index ad39f8b..18694cf 100644
--- a/libs/defaults.h
+++ b/libs/defaults.h
@@ -153,7 +153,7 @@
 #define BROKEN_MINSIZE_LIMIT 30000
 
 /* geometry window */
-#define GEOMETRY_WINDOW_BW                 2 /* pixels */
+#define GEOMETRY_WINDOW_BW                 1 /* pixels */
 #define GEOMETRY_WINDOW_STRING             " +8888 x +8888 "
 #define GEOMETRY_WINDOW_POS_STRING         " %+-4d %+-4d "
 #define GEOMETRY_WINDOW_SIZE_STRING        " %4d x %-4d "
diff --git a/modules/FvwmProxy/FvwmProxy.c b/modules/FvwmProxy/FvwmProxy.c
index c5f1d4c..5c74fb7 100644
--- a/modules/FvwmProxy/FvwmProxy.c
+++ b/modules/FvwmProxy/FvwmProxy.c
@@ -1225,7 +1225,7 @@ static void DrawWindow(
 	{
 		XClearWindow(dpy,proxy->proxy);
 	}
-	RelieveRectangle(dpy,proxy->proxy, 0,0, w - 1,h - 1, hi_gc,sh_gc, 2);
+	RelieveRectangle(dpy,proxy->proxy, 0,0, w - 1,h - 1, hi_gc,sh_gc, 1);
 
 #if 0
 	if (proxy->iconname != NULL)
-- 
2.0.5

